.circleci/config.yml

```diff
@@ -30,6 +30,14 @@ jobs:
     
     steps:
       - checkout # check out the code in the project directory
+      - run: 
+          name: Build Dietstory Django Prod
+          command: |
+            cd dietstory-routing && docker build -t benjixd/dietstory -f Dockerfile.prod .
```

Comment:
Build this docker image only during deploy stages. We need to copy the `dietstory_secret_config.json` and copy that including the python project to the docker image. See `s3://dietstory-api-server-assets/{env}/config/`

Reply:
Note that we need to build a dev and prod image.

Reply:
Added a steps to pull the secret config file from S3, use it to build our docker images and then delete the file. We also need a step to push to docker hub, but maybe we should create a dietstory account for that? Otherwise you'd have to put your login and password.

Reply:
Yes this can be arranged